setup.py

```diff
@@ -35,5 +35,5 @@ def package_data(pkg, root_list):
     entry_points={
         'xblock.v1': 'drag-and-drop-v2 = drag_and_drop_v2:DragAndDropBlock',
     },
-    package_data=package_data("drag_and_drop_v2", ["static", "templates", "public"]),
+    package_data=package_data("drag_and_drop_v2", ["static", "templates", "public", "translations"]),
```

Comment:
In order to update the version used in edx-platform, you'll have to do a version bump yes - see previous version bump PRs, because you have to update a few separate things - I think setup.py, circle.yml, and Changelog.md